lib/views/general/_responsive_topnav.html.erb

```diff
@@ -1,31 +1,39 @@
 <div id="topnav" class="topnav">
-  <ul id="navigation" class="navigation" >
+  <ul id="navigation" class="navigation" role="navigation">
 
     <li class="<%= 'selected' if params[:controller] == 'request' and ['new', 'select_authority'].include?(params[:action]) %>">
       <%= link_to _("Make a request"), select_authority_path, :id => 'make-request-link' %>
     </li>
 
     <li class="<%= 'selected' if params[:controller] == 'request' and !['new', 'select_authority'].include?(params[:action]) %>">
-      <%= link_to _("Browse requests"), request_list_all_path %>
+      <%= link_to _("Browse requests"), request_list_successful_path %>
     </li>
 
     <li class="<%= 'selected' if params[:controller] == 'public_body' %>">
       <%= link_to _("View authorities"), list_public_bodies_default_path %>
     </li>
 
+    <% unless AlaveteliConfiguration::blog_feed.empty? %>
+      <li class="<%= 'selected' if params[:controller] == 'general' and params[:action] == 'blog' %>"><%= link_to _("Read blog"), blog_path %></li>
+    <% end %>
+
 
     <li><%= link_to "Chi Siamo", ' http://www.dirittodisapere.it/chi-siamo/' %></li>
     <li class="<%= 'selected' if params[:controller] == 'help' %>">
       <%= link_to _("Help"), help_about_path %>
     </li>
 
-    <li class="navigation_search">
-      <form class="navigation_search_form" method="get" action="<%= search_redirect_path %>" role="search">
-        <label class="visually-hidden" for="navigation_search_input">
+    <li id="navigation_search" class="navigation_search">
+      <form id="navigation_search_form" class="navigation_search_form" method="get" action="<%= search_redirect_path %>" role="search">
+        <label class="visually-hidden" for="navigation_search_button">
           <%= _("Search") %>
         </label>
-        <%= text_field_tag 'query', params[:query], { :id => "navigation_search_input", :type => "search", :placeholder => _("Search"), :title => _("type your search term here") } %>
-        <button type="submit"><%= _("Search") %></button>
+        <%= text_field_tag 'query', params[:query], { :class => "navigation_search_button", :id => "navigation_search_button", :type => "search", :placeholder => _("Search"), :title => _("type your search term here") } %>
+        <button type="submit">
```

Comment:
The button has lost its green background, making the search icon invisible:

**Before**

![screen shot 2017-01-09 at 12 20 39](https://cloud.githubusercontent.com/assets/282788/21766385/10ce7982-d666-11e6-86e6-f5faff60fa0d.png)


**After**

![screen shot 2017-01-09 at 12 20 00](https://cloud.githubusercontent.com/assets/282788/21766377/fdb162ce-d665-11e6-851a-7ebc226874eb.png)